contracts/Pool/PoolRegistry.sol

```diff
@@ -108,7 +109,7 @@ contract PoolRegistry is OwnableUpgradeable {
     /**
     * @dev Maps comptroller address to Venus pool Index.
     */
-    mapping(address => uint256) private _poolByComptroller;
+    mapping(address => VenusPool) private _poolByComptroller;
```

Comment:
@coreyar current version of code, had this storage mapping to lookup `poolId` from `comptrollerAddress`
in existing code its mapping for poolId -> VenusPoolStruct
should we retain same way ?


now that this PR is to use `poolId` as primary reference while looking-up the Pool
and mapping has comptrollerAddress -> VenusPool Struct